src/cosmic_ray/cli.py

```diff
@@ -185,10 +195,20 @@ def handle_baseline(args):
                 end_pos=template.end_pos,
                 job_id=template.job_id))
 
-    # Run the single-entry session.
-    cosmic_ray.commands.execute(baseline_session_file)
+        # Run the single-entry session.
+        cosmic_ray.commands.execute(db)
 
-    return ExitCode.OK
+        result = next(db.results)[1]  # type: WorkResult
+        if result.test_outcome == TestOutcome.KILLED:
+            if dump_report:
+                print("Execution with no mutation gives those following errors:")
+                for line in result.output.split('\n'):
+                    print("  >>>", line)
+            return 1
```

Comment:
You should use an appropriate enum from ExitCode here.

Reply:
`cosmic-ray baselines` means: "Does my program runs correctly inside cr system ?"
Return code convention wants that possible answers will be: 0 for yes or 1 for no.
All others return codes conventions are to tells "I can't answer correctly because ...".
Then, for me, there is no better returns that 1.

Reply:
OK, I can live with this.